tests/integration/archiveTechRecordStatus.intTest.ts

```diff
@@ -19,16 +19,15 @@ describe("archiveTechRecordStatus", () => {
   beforeAll(async () => {
     jest.restoreAllMocks();
     // await emptyDatabase();
-    await populateDatabase();
+    // await populateDatabase();
```

Comment:
do we need this commented out line?

Reply:
I think I just missed some on when I was deleting the others